src/mmw/js/src/user/views.js

```diff
@@ -385,10 +387,37 @@ var ItsiSignUpModalView = ModalBaseView.extend({
     }
 });
 
+var ChangePasswordModalView = ModalBaseView.extend({
+    template: changePasswordModalTmpl,
+
+    ui: _.defaults({
+        'oldPassword': '#old_password',
+        'password1': '#new_password1',
+        'password2': '#new_password2'
+    }, ModalBaseView.prototype.ui),
+
+    onModalShown: function() {
+        this.ui.oldPassword.focus();
+    },
+
+    onValidationError: function() {
+        this.ui.oldPassword.focus();
+    },
+
+    setFields: function() {
+        this.model.set({
+            old_password: $(this.ui.oldPassword.selector).val(),
+            new_password1: $(this.ui.password1.selector).val(),
+            new_password2: $(this.ui.password2.selector).val(),
+        }, { silent: true });
```

Comment:
Why the `silent`? Is anything listening to this that shouldn't be tripped?

Reply:
Great q. I followed the pattern the other user views were using, and just assumed the `silent` flag was to prevent `validate` firing on change. [Reading the docs now](http://backbonejs.org/#Model-validate) seems that's not the case. Going to remove this, see what happens, and get back to you.

Reply:
Not sure why this is on the other views: https://github.com/WikiWatershed/model-my-watershed/commit/ec2dd6434434704051496a33513f34c41d115b5e#diff-bbc329caa6aad1d9e2bf745db301390fR43

Removed it from this new one and it worked fine. Suspect it's not needed on the others, but don't want to mess around right now. Will push a fixup that removes it from the new view, but leaves the others.

Reply:
There's probably a `silent` there because the view [is listening to the model's changes](https://github.com/WikiWatershed/model-my-watershed/commit/ec2dd6434434704051496a33513f34c41d115b5e#diff-bbc329caa6aad1d9e2bf745db301390fR30) and we don't want to re-render the view upon validation.

Reply:
All those views listen as well https://github.com/WikiWatershed/model-my-watershed/blob/5e2385a8f932ff270b006e59af57ae0d0d491610/src/mmw/js/src/user/views.js#L42. Going to keep the `silent` then.

Reply:
🤐